packages/fluentui/react-northstar/src/themes/teams/components/Breadcrumb/breadcrumbDividerStyles.ts

```diff
@@ -0,0 +1,11 @@
+import { pxToRem } from '../../../../utils';
+import { ComponentSlotStylesPrepared, ICSSInJSStyle } from '@fluentui/styles';
+import { BreadcrumbDividerStylesProps } from '../../../../components/Breadcrumb/BreadcrumbDivider';
+
+export const breadcrumbDividerStyles: ComponentSlotStylesPrepared<BreadcrumbDividerStylesProps, {}> = {
+  root: ({ props: p, variables: v }): ICSSInJSStyle => {
+    return {
+      margin: `0 ${pxToRem(8)}`,
```

Comment:
```suggestion
  root: ({ props: p, variables: v }): ICSSInJSStyle => {
    return {
      margin: v.dividerMargin,
```

Let's extract this to a variable

Reply:
fixed by commit [feat(Breadcrumb): extract style variable](https://github.com/microsoft/fluentui/pull/14926/commits/e1bbef872ed494f842153450a4bff21ca5878f0b)